winum.el

```diff
@@ -447,6 +449,7 @@ windows, however a higher number can be reserved by the user-defined
 
 (defun winum--window-list ()
   "Return a list of interesting windows."
+  (winum--check-for-scope-change)
```

Comment:
I don't think we need that one, this function doesn't interact at all with the internal variables. It just builds an ordered list of the existing emacs windows that we will number afterwards.